docs/configure/toolbar.md

```diff
@@ -0,0 +1,43 @@
+---
+title: 'Toolbar'
+---
+
+<YouTubeCallout id="TODO?" title="How to Configure the Toolbar" />
+
+Storybook’s toolbar appears at the top of stories and docs pages. It displays menus from addons that can customise the preview environment, like the [viewport] and [background]) addons, as well as [custom menus]() you've defined.
+
+### Conditionally show toolbar
+
+By default, the toolbar is visible on every page, and users can choose to hide it with a keyboard shortcut.
+
+If you need, you can customise when the toolbar appears by adding the following configuration:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'common/storybook-manager-hide-toolbar-on-docs.js.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+The following parameters are available to the `showToolbar` function:
+
+
+| Name                     |     Type      |                           Description                            |            Possible Value             |
+| ------------------------ | :-----------: | :--------------------------------------------------------------: | :-----------------------------------: |
+| **path**                 |    String     |                 Path to the page being displayed                 |  `/story/components-button--default`  |
```

Comment:
If the value is a string, then that should be represented here:

```suggestion
| **path**                 |    String     |                 Path to the page being displayed                 |  `'/story/components-button--default'`  |
```

Applies to all `String` type possible values.

Reply:
Thanks, I wasn't sure about that. I inspired myself from existing docs, so is it expected that some docs show strings without single quotes? Is it just debt?

Reply:
Oh, apparently. 😅 

If you point out the ones you're aware of to me, I'll make sure they're addressed. 🙏

Reply:
* docs/addons/addons-api.md
* docs/configure/features-and-behavior.md
* docs/essentials/viewport.md (device type)
* docs/writing-stories/naming-components-and-hierarchy.md (locale)

Reply:
I've used a quick and dirty code search, could have missed some obviously.